src/likelihoodratiotest.jl

```diff
@@ -46,12 +46,24 @@ Base.getindex(lrt::LikelihoodRatioTest, s::Symbol) = getfield(lrt,s)
 
 """
     likelihoodratiotest(m::MixedModel...)
+    likelihoodratiotest(m0::LinearModel, m::MixedModel...)
+    likelihoodratiotest(m0::GeneralizedLinearModel, m::MixedModel...)
+    likelihoodratiotest(m0::TableRegressionModel{LinearModel}, m::MixedModel...)
+    likelihoodratiotest(m0::TableRegressionModel{GeneralizedLinearModel}, m::MixedModel...)
+
 
 Likeihood ratio test applied to a set of nested models.
 
-Note that nesting of the models is not checked.  It is incumbent on the user
-to check this. This differs from [`StatsModels.lrtest`](@ref) as nesting in
-mixed models, especially in the random effects specification, may be non obvious.
+!!! note
+    The nesting of the models is not checked.  It is incumbent on the user
+    to check this. This differs from [`StatsModels.lrtest`](@ref) as nesting in
+    mixed models, especially in the random effects specification, may be non obvious.
+
+!!! note
+    For comparisons between mixed and non-mixed models, the deviance for the non-mixed
+    model is taken to be -2 log likelihood, i.e. omitting the additive constant for the
+    fully saturated model. This is in line with the computation of the deviance for mixed
+    models.
```

Comment:
@dmbates I can feel you looking over my shoulder and shaking your head when I write something like "the deviance  isn't really the deviance".  :slightly_smiling_face:  Do you have a better way of handling / labelling this?

Reply:
The important point is that the difference in -2 log likelihood is the same as the difference in deviances would be if they could be calculated.  Of course, now that I have written that I wonder if it makes sense when comparing a LinearModel and a LinearMixedModel.  We know what the saturated model for a LinearModel is but we don't for a LinearMixedModel.  My statement seems to indicate that the deviance for the saturated model in a LinearMixedModel would then have to be the same as that of a LinearModel but I am not sure that makes sense.  I am starting to confuse myself.

Reply:
I'm glad I'm not the only one who struggles with some of this stuff. :smile: 

I think the relevant bit here is actually that the test statistic for LRT is -2 log(L_1/L2) which then is just the difference of -2 log likelihood. Using the true deviance was always just a convenience because the additive constants cancel out, when both models have the same constant.

All that said, my question is: should we change the label of that column and/or the internal fields? I would prefer to keep the internal fieldnames, but changing the `show` method's output is trivial.

Reply:
Hmm. For a linear mixed model it would be more accurate to label the column as `-2 loglik`.  For a GLMM I think it should be `deviance`.  I believe that for a `Binomial` GLMM that is not also `Bernoulli` (e.g. `cbpp`) the quantity in that column will not be `-2 loglik`

Reply:
Ah, yes, that is true. And indeed the `loglikelihood` and `deviance` for GLMM are defined independently. 

Let me see what it would take to add a conditional to the `show` method --I think we'll need to add something to the `LikelihoodRatioTest` struct because right now there is no notion of which model it came from.